src/software/backend/handheld_controller_backend.cpp

```diff
@@ -0,0 +1,40 @@
+#include "software/backend/handheld_controller_backend.h"
+
+#include "software/time/timestamp.h"
+#include "software/util/design_patterns/generic_factory.h"
```

Comment:
Why do we have a backend for this? I would expect to use the radio backend, and have the controller replace the AI?

Reply:
The We need a separate backend for this (for now) because we need to spoof camera data so the robot state doesn't drift. Because the `DirectVelocity` primitive still uses the internal estimated state of the robot to apply velocity, not having vision causes this internal state to drift and therefore the control to drift. The backend spoofs some camera data to prevent this drifting.

I have created #1223 and #1224 to address the issues with `DirectVelocity` and to remove the `HandheldControllerBackend` afterwards. Let me know if this makes sense.